nservicebus/testing/index_content_testing_[6,).partial.md

```diff
@@ -18,14 +40,29 @@ The following unit test checks that the total amount has the discount applied:
 
 snippet: SagaTest
 
+
 ## Testing a behavior
 
+[Message pipeline behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md) also can be tested, but using a different testable context objects. Each of the [various pipeline stages](/nservicebus/pipeline/steps-stages-connectors.md) uses a specific interface for its context, and each context interface has a testable implementation following a name convention.
```

Comment:
using ~~a~~ different testable context objects.

following a ~~name~~ naming convention

Reply:
Actually, drop "following a name convention." completely. It's not needed because of the next section.